estate/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+from . import estate_property
+from . import estate_property_tag
+from . import estate_property_offer
+from . import estate_property_type
+from . import estate_user
```

Comment:
```suggestion
from . import res_users
```
This is an override of an existing class, so it should match its name. `res.users` -> res_users.py